packages/codemods/src/v5/is-loading/__testfixtures__/default-import.input.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react'
```

Comment:
@TkDodo does it cover most of the use cases? Feel free to provide a few if you think it lacks some of possible use cases. :)

Reply:
👍